docs/DeveloperGuide.md

```diff
@@ -672,6 +672,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### **1. More Specific Success Message for Company**
 
+**Feature Flaw in Current Implementation**
+
+Currently, the success message for view, edit, remark, unremark, and add commands only displays the company name.
+However, the duplicate check implemented in LinkMeIn uses 3 criteria, company name, role, and deadline. As such, there
+can be more than one entry with the same company name. The user might want to know the role and deadline to
+differentiate between applications with the same company name. As such, with the current implementation of the success
+message, it can be confusing for the user as to which company specifically has been modified.
```

Comment:
As such, the current success message displayed can be confusing for the user as to which company specifically has been modified.